tensorflow/c/eager/c_api_test_util.h

```diff
@@ -53,6 +56,27 @@ TFE_TensorHandle* TestTensorHandleWithDimsFloat(TFE_Context* ctx, float data[],
 TFE_TensorHandle* TestTensorHandleWithDimsInt(TFE_Context* ctx, int data[],
                                               int64_t dims[], int num_dims);
 
+// Return a tensor handle with given type, values and dimensions.
+template <class T, TF_DataType datatype>
+TFE_TensorHandle* TestTensorHandleWithDims(TFE_Context* ctx, const T* data,
+                                           const int64_t* dims, int num_dims) {
+  TF_Status* status = TF_NewStatus();
+  TF_Tensor* t = TFE_AllocateHostTensor(ctx, datatype, dims, num_dims, status);
+  memcpy(TF_TensorData(t), data, TF_TensorByteSize(t));
+  TFE_TensorHandle* th = TFE_NewTensorHandleFromTensor(ctx, t, status);
+  CHECK_EQ(TF_OK, TF_GetCode(status)) << TF_Message(status);
+  TF_DeleteTensor(t);
+  TF_DeleteStatus(status);
+  return th;
+}
+
+// Return a scalar tensor handle with given values.
+template <class T, TF_DataType datatype>
+TFE_TensorHandle* TestScalarTensorHandle(TFE_Context* ctx, const T value) {
```

Comment:
Could we update other helpers in c_api_test_util.cc to use these new methods?

Reply:
Done

Reply:
Thanks for the cleanup. I was wondering if we can further use this to implement other functions in _this_ file e.g. https://github.com/tensorflow/tensorflow/blob/11ab65610238ab8469dd0a4c183ca4dc272ef61a/tensorflow/c/eager/c_api_test_util.cc#L108-L120

Reply:
I did notice this file and other files in `c/eager`. But I am not sure if it will break something. @kkimdev What do you think ?

Reply:
@saxenasaurabh , Since @kkimdev hasn't replied yet, could we merge this PR and I will send another PR if needed ?

Reply:
I don't think this will break anything. Also these are test utilities so this is less risky of a refactor. Doing this as a follow up sounds good.